benchmarks/singleshot/singleshot.lp

```diff
@@ -0,0 +1,9 @@
+#include "ooasp.lp".
+#include "ooasp_symmetry.lp".
```

Comment:
For better comparability, please keep #include statements that already exist in ooasp.lp there, and only add includes here that do not exist in the multi-shot version of ooasp.lp